persistence-api/src/main/java/io/stargate/db/Persistence.java

```diff
@@ -34,6 +34,9 @@
  */
 public interface Persistence {
 
+  int SCHEMA_AGREEMENT_WAIT_RETRIES =
```

Comment:
I think that it will be more user-friendly to have one global timeout setting and then split it into iterations, where every iteration lasts `200ms`.
Right now, to understand how this parameter translates to the actual timeout you need to `200 * SCHEMA_AGREEMENT_WAIT_RETRIES`, wdyt?

Reply:
Sure, but I also think we ought to clean-up the "wait for agreement API" in more substantial ways. Ideally, I think the timeout should be delegated to the caller. I'll file an issue for this.

Reply:
#318